### DISCUSSION TITLE
Welcome to YAOG Discussions!

### DISCUSSION THREAD

**Discussion by patrickpr** [patrickpr/YAOG#11] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KJTR** [comment]
Is there anyway I could donate to you patrickpr for making this? Every time i use openssl via console to make a private key or crt file or a pfx file it says the password is wrong even when the test password is "test". When i use your tool i don't have these issues. Thank you for making this. Is there anyway to add a new feature to link the private key together with a crt file to generate a PFX for self signing executables?
